ui/src/components/queries/AggregationQuery.js

```diff
@@ -24,30 +22,28 @@ const fetchAggregationData = async ({ apiFetcher, field, sqon }) => {
       }
     }`;
 
-  const { data } = await apiFetcher({
+const AggregationQuery = ({ sqon, ...props }) => {
+  const { field: fieldName } = props;
```

Comment:
for a later PR: it's worth renaming "field" up the tree, rather than just renaming locally in multiple places.

Reply:
Something about this might have broken the Aggregations visuals, revisiting

Reply:
OK found I place I hadn't updated to fieldName, fixed now